src/admin/client.ts

```diff
@@ -74,7 +76,7 @@ class DisplayOrder {
 
     public submit = () => {
         var msg = new Models.OrderRequestFromUI(this.side, this.price, this.quantity, this.timeInForce, this.orderType);
-        this._log.info("submitting order", msg);
```

Comment:
this is not related to the full space of the vmdk of docker, but instead is to minimize the traffic of the application. I preffer the app to send data to the exchange api rathen than to the client (if that msg to the client is just a notice of an action already known by the client by other means [rich gui]).

Also, users do not look at the console; if this is for devs, let devs activate (uncomment) this stream of debuggin data when needed rather than force all users to transmit it..

if you dont agree i can uncomment it in a next commit easy  :blush: